pkg/config/config.go

```diff
@@ -101,6 +101,7 @@ func Labels() *cli.StringSlice {
 		"branch",
 		"number",
 		"run",
+		"actor",
```

Comment:
This shouldn't be enabled by default, this will increase the labels pretty massively and should only be enabled per instance.

Reply:
Yes, that way